tox.ini

```diff
@@ -0,0 +1,67 @@
+[tox]
+envlist = py{35,36,37}-{pytest,setup}{,-all,-dev},docs,flake8
+skipsdist = True
+requires = tox-venv
+           setuptools >= 30.0.0
+
+[testenv]
+usedevelop = True
+whitelist_externals = which
+passenv = HOME LC_ALL LC_CTYPE CC CFLAGS
+description =
+    pytest: run tests with pytest,
+    setup: run tests with python setup.py,
+    all: and all dependencies,
+    dev: and numpy and matplotlib dev versions,
+# declaring numpy and pytest is needed for --force-dep
```

Comment:
Just to check, is this still needed since --force-dep isn't used?

Reply:
Yes indeed, I removed this comment.